prow/cmd/hook/githubagent.go

```diff
@@ -89,6 +91,9 @@ func (ga *GitHubAgent) handleIssueCommentEvent(ic github.IssueCommentEvent) {
 	if err := ga.commentTrigger(ic); err != nil {
 		l.WithError(err).Error("Error triggering after issue comment event.")
 	}
+	if err := ga.lgtmComment(ic); err != nil {
```

Comment:
As discussed, I'm not a huge fan of the way we dispatch events :-). While I would like this to be addressed quickly (probably before we put to much effort into migrating more commands), I think it should be fine for this PR.